feather/server/config.toml

```diff
@@ -13,6 +13,8 @@ motd = "A Feather server"
 max_players = 16
 default_gamemode = "creative"
 view_distance = 12
+default_header = ""
+default_footer = ""
```

Comment:
Users won't understand what header/footer is, I think it should be named like `tablist_header` or `playerlist_header`, and have default values to make it even more obvious (like vanilla's "A Minecraft Server" default motd).